HEATDamage.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Text;
+using System.Diagnostics;
+
+namespace PhysicalDamage.Core
+{
+    class HEATDamage:Damage
+    {
+        public struct HEATData
+        {
+            public float Pierce;
+
+            public float ArmorHEATFactor;
+
+            public float HealthHEATFactor;
+        }
+
+        private HEATData data;
```

Comment:
Convention for private variables is use to `_`. Rename to `_heatData` or `_data`.